service.js

```diff
@@ -4,7 +4,7 @@ let debugZmqTx = require('debug')('service:zmq:tx')
 let Indexd = require('indexd')
 let leveldown = require('leveldown')
 let rpc = require('./rpc')
-let zmq = require('zmq')
+let zmq = require('zeromq')
```

Comment:
why change this?

Reply:
(I mean,  seams like a good change,  but,  why?)

Reply:
check `zmq` repo page: https://github.com/JustinTulloss/zeromq.node
project is moved to `zeromq`